Yoast/wordpress-seo
# Made kb-search.js deprecated and removed related styles and references

## Summary

This PR can be summarized in the following changelog entry:

* Cleaned up codebase, by removing old kb-search code.

## Relevant technical choices:

* Made `wp-seo-kb-search.js` deprecated, removed stylesheets and enqueues related to `kb-search`.

## Test instructions

This PR can be tested by following these steps:

* Navigate various pages where the help center exists ( Yoast SEO admin pages, edit pages, edit posts).
* Make sure to check the console, and make sure no unexpected errors occur.

Fixes #7946
